MANIFEST.in

```diff
@@ -8,4 +8,5 @@ exclude tasks.py
 exclude test/test_fixtures/*/*.json
 exclude test/test_hpack_integration.py
 include README.rst LICENSE CHANGELOG.rst CONTRIBUTORS.rst tox.ini Makefile
+include src/hpack/py.typed
```

Comment:
I'm still trying to fully understand all the moving parts, but from https://packaging.python.org/guides/using-manifest-in/ I would argue that this line is not necessary, because we already have `py.typed` in the `package_data`.

Reply:
I don't think package_data 
```suggestion
```